bin/installer.sh

```diff
@@ -225,7 +225,7 @@ function check_snmptrapd_run() {
 	echo
 	
 	# Check port 
-	port=$(netstat -apn |grep -E "^udp.*:162" 2>&1)
+	port=$(ss -pluna | grep ':162 ' 2>&1)
```

Comment:
Same as before and get only one line (you often get two with ipv4 and ipv6 listeners).

```suggestion
	port=$(ss -plun | grep ':162 ' | head -1 2>&1)
```